### DISCUSSION TITLE
Playlist bulk download

### DISCUSSION THREAD

**Discussion by Navarro21a** [mediacms-io/mediacms#1026] [Q&A]
Hi Folks, 

Just got this intsalled and testing it out, is there a way to enable bulk download from a playlist?  I'ld like to be able to put all my choices in a playlist and then grab the original files in one download.  Possible?

Cheers

**Comment by Freigeist069** [comment]
Hi there,

I don't think there is a feature like that. If you don't find any information [here](https://github.com/mediacms-io/mediacms/blob/main/docs/admins_docs.md), I recommend writing a feature request.

Best, Joerg

**Reply by Navarro21a** [reply]
Cool, I'll do that.  I haven't seen any reference to it.
